src/query/catalog/src/plan/partition.rs

```diff
@@ -185,16 +185,22 @@ impl Default for Partitions {
 pub struct StealablePartitions {
     pub partitions: Arc<RwLock<Vec<VecDeque<PartInfoPtr>>>>,
     pub ctx: Arc<dyn TableContext>,
+    pub disable_steal: bool,
```

Comment:
This option is used for?

Reply:
It's to improvement `topk` queries, we should disable steal partitions from other threads.

If parts goes into topk in specific order, it could have stable and fastest performance.

Reply:
It would be better add some comments for it.